packages/consumption/test/modules/requests/itemProcessors/deleteAttribute/DeleteAttributeRequestItemProcessor.test.ts

```diff
@@ -684,7 +685,113 @@ describe("DeleteAttributeRequestItemProcessor", function () {
     });
 
     describe("applyIncomingResponseItem", function () {
-        test("sets the deletionInfo of an own shared Identity Attribute", async function () {
+        test("doesn't change the deletionInfo if a simple AcceptRequestItem is returned", async function () {
+            const deletionDate = CoreDate.utc().subtract({ days: 1 });
+            const deletedByPeerAttribute = await consumptionController.attributes.createAttributeUnsafe({
+                content: IdentityAttribute.from({
+                    value: {
+                        "@type": "BirthName",
+                        value: "Schenkel"
+                    },
+                    owner: accountController.identity.address
+                }),
+                shareInfo: {
+                    peer: peerAddress,
+                    requestReference: CoreId.from("reqRef")
+                },
+                deletionInfo: {
+                    deletionStatus: DeletionStatus.DeletedByPeer,
+                    deletionDate: deletionDate
+                }
+            });
+
+            const requestItem = DeleteAttributeRequestItem.from({
+                mustBeAccepted: false,
+                attributeId: deletedByPeerAttribute.id
+            });
+
+            const requestId = await ConsumptionIds.request.generate();
+            const incomingRequest = LocalRequest.from({
+                id: requestId,
+                createdAt: CoreDate.utc(),
+                isOwn: false,
+                peer: peerAddress,
+                status: LocalRequestStatus.DecisionRequired,
+                content: Request.from({
+                    id: requestId,
+                    items: [requestItem]
+                }),
+                statusLog: []
+            });
+
+            const responseDeletionDate = CoreDate.utc().add({ days: 1 });
+            const responseItem = DeleteAttributeAcceptResponseItem.from({
```

Comment:
This should be just an AcceptResponseItem and not a DeleteAttributeAcceptResponseItem as the test is for simple AcceptRequestItems that are returned.